app/Http/Controllers/AccessoriesController.php

```diff
@@ -296,14 +331,21 @@ public function postCheckout(Request $request, $accessoryId)
     public function getCheckin(Request $request, $accessoryUserId = null, $backto = null)
     {
         // Check if the accessory exists
-        if (is_null($accessory_user = DB::table('accessories_users')->find($accessoryUserId))) {
-            // Redirect to the accessory management page with error
-            return redirect()->route('accessories.index')->with('error', trans('admin/accessories/message.not_found'));
+        if ($accessory_user = DB::table('accessories_users')->find($accessoryUserId)) {
```

Comment:
While we're in here I would definnitely love it if we could Laravel-ize some of these `DB::table()` calls - but this is more of a 'nice to have' rather than 'you must do it' kind of thing.